custom/icds_reports/reports/fact_sheets.py

```diff
@@ -528,38 +561,21 @@ def new_table_config(self):
         return fact_sheet_data_config
 
     def data_sources(self, config):
-        return {
-            'AggChildHealthMonthlyDataSource': AggChildHealthMonthlyDataSource(
-                config=config,
-                loc_level=self.loc_level,
-                show_test=self.show_test,
-                beta=self.beta
-            ),
-            'AggCCSRecordMonthlyDataSource': AggCCSRecordMonthlyDataSource(
-                config=config,
-                loc_level=self.loc_level,
-                show_test=self.show_test,
-            ),
-            'AggAWCMonthlyDataSource': AggAWCMonthlyDataSource(
-                config=config,
-                loc_level=self.loc_level,
-                show_test=self.show_test,
-                beta=self.beta
-            )
+        fact_sheet_datasources = dict()
+        datasource_arguments = {
+            'config': config,
+            'loc_level': self.loc_level,
+            'show_test': self.show_test
         }
+        for datasource_name, datasource_class in FACT_SHEET_DATASOURCES_NAMES.items():
+            arguments = datasource_arguments.copy()
+            if datasource_name != 'AggCCSRecordMonthlyDataSource':
```

Comment:
the fact that it's repeated here makes me like the idea of adding it even more

Reply:
yeah I think that makes more sense, I can add the beta param to the this class as well